osu.Game/Beatmaps/Drawables/UpdateableBeatmapBackgroundSprite.cs

```diff
@@ -47,16 +47,17 @@ protected override Drawable CreateDrawable(BeatmapInfo model)
         {
             Drawable drawable;
 
-            var localBeatmap = beatmaps.GetWorkingBeatmap(model);
+            if (model?.OnlineBeatmapID != null)
```

Comment:
isn't this in the wrong place? shouldn't it be inside the `else if`? 

99% sure this is going to regress behaviour as the database doesn't provide `OnlineInfo` and you are now forcing usage of the database beatmap.

Reply:
We have to query for the beatmap here though, or else we won't know whether or not there is an actual local version (the original issue we had). This is why I had the localinfo variable earlier.

Reply:
Nevermind that, I can find a different solution